### DISCUSSION TITLE
line spacing in float captions too narrow?

### DISCUSSION THREAD

**Discussion by timmli** [langsci/latex#140] [Q&A]
Using a recent skeleton, the line spacing in float captions looks very/too narrow to me. Consider the tiny distance between the `g` and the `M` in the following example:

![unbenannt](https://user-images.githubusercontent.com/4593031/28456512-dfdb9546-6e02-11e7-82f2-26029c1bac6d.PNG)

**Comment by Glottotopia** [comment]
there might be a point in having `{stretch=.9}` instead of `{stretch=.8}` for `\captionsetup`

**Comment by stefan11** [comment]
Yes, please. This is related: https://github.com/langsci/259/issues/42
